### ISSUE TITLE
Empty type in interface if `type` is missing from *.settings.json

### ISSUE THREAD

**Issue by ShahNafis** [erikvullings/strapi-to-typescript#9]
Hello!

I noticed an error in the output of the interface on two of my models. The model I have is called `Aboutme` and this is the `aboutme.settings.json`

```json
{
  "kind": "singleType",
  "collectionName": "aboutmes",
  "info": {
    "name": "Aboutme",
    "description": ""
  },
  "options": {
    "increments": true,
    "timestamps": true,
    "draftAndPublish": true
  },
  "attributes": {
    "text": {
      "type": "richtext"
    },
    "avatar": {
      "model": "file",
      "via": "related",
      "allowedTypes": [
        "images"
      ],
      "plugin": "upload",
      "required": false
    }
  }
}
```

The produced output is
```js
export interface IAboutme {
  id: string;
  text?: string;
  avatar?: ;
}
````

If you look at the `avatar` field it is missing the type. For this example, `avatar` is an image field (single image).

I noticed it also happens for files(single file), for example, my `Resume` model

```json
{
  "kind": "singleType",
  "collectionName": "resumes",
  "info": {
    "name": "Resume",
    "description": ""
  },
  "options": {
    "increments": true,
    "timestamps": true,
    "draftAndPublish": true
  },
  "attributes": {
    "pdf": {
      "model": "file",
      "via": "related",
      "allowedTypes": [
        "files"
      ],
      "plugin": "upload",
      "required": false
    },
    "date": {
      "type": "date"
    }
  }
}
```

Output

```js
export interface IResume {
  id: string;
  pdf?: ;
  date?: Date;
}
```

Hopefully this makes sense or is useful!

**Comment by aperron** [comment]
Thanks you for your feedback.

The model File is defined in the Strapi download plugin. You can add this definition using multiple input: `sts ./api ./node_modules/strapi-plugin-upload/models/ -g ./components/ -o ./sts`.

Normally, if the model is not known, it's fallback on `any`. I'll fixed this.

**Comment by aperron** [comment]
bug fixed in v2.0.1

**Comment by ShahNafis** [comment]
Hello, 

Both ways work now, thank you very much. I didn't know I had to add in the upload plugin path.
